### ISSUE TITLE
corrupted_text question should invalidate commands operating on README

### ISSUE THREAD

**Issue by xiaket** [jarv/cmdchallenge#19]
Apparently, the following command is accepted, while it shouldn't be.

```
head -n 15 README | tail -n 9 | sed "s/^# //g"
```

**Comment by joshtriplett** [comment]
I admit to doing the same, because the specifics of which '!' characters to delete seems so fiddly that I'd just use a text editor for that one rather than a set of regexes.

**Comment by 9Yg1rxeSeha90ZU1** [comment]
yeah. solved this with multiple sed parameters but still even after validating manually this challenge didnt get accepted by validator. seems the more recent challenges are a bit buggy yes?
